airgun/entities/discoveryrule.py

```diff
@@ -0,0 +1,101 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.discoveryrule import (
+    DiscoveryRuleCreateView,
+    DiscoveryRuleEditView,
+    DiscoveryRulesView,
+)
+from airgun.views.host import HostsView
+
+
+class DiscoveryRuleEntity(BaseEntity):
+
+    def create(self, values):
+        view = self.navigate_to(self, 'New')
+        view.fill(values)
+        view.submit.click()
+
+    # TODO use entity names, after the bug in widgetastic will be solved
+    def delete(self, entity_name):
+        view = self.navigate_to(self, 'All')
+        view.table.row()[6].widget.fill('Delete')
```

Comment:
I know that is an insane request, but just in case you interested. Personally, I don't like to see indexes here. Don't you mind to raise PR in foreman to change empty header to `Actions` or whatever:
https://github.com/theforeman/foreman_discovery/blob/develop/app/views/discovery_rules/index.html.erb

Let me know your decision

Reply:
To `<th><%= _("Actions") %></th>` or similar

Reply:
Okay, I can try it, why not. I agree with your idea.  After I will come back from my time off I will do that.